examples/dgfip_c/m_error.h

```diff
@@ -14,12 +14,17 @@ typedef struct m_error {
   char minor_code[7];
   char description[81];
   char isisf[2];
-  bool has_occurred;
+  char code_information[10];
```

Comment:
Maybe document that the sized alloted for the description and the code information depend on the actual strings present in the M codebase and are susceptible to cause bugs if overflowed?

Reply:
How would you feel about turning them into `const char *` instead ? 
All m_error structs are static in code and therefore don't need a specific size. This also avoids the overflowing problem.